core-3/src/main/java/configuration/engine/FourCylinderEngine.java

```diff
@@ -0,0 +1,12 @@
+package configuration.engine;
+
+import org.springframework.stereotype.Component;
+
+@Component("oldSchoolStuff")
```

Comment:
The same as the "Bumblebee" bean, it is not necessary since I am already defining the Bean in the java bean configuration file. I will remove it and keep the bean definition in the configuration.